src/Core/CoreKernel.php

```diff
@@ -209,7 +209,7 @@ protected function bootDatabaseGlobals()
         global $databaseConfig;
         global $database;
 
-        $prefix = getenv('SS_DATABASE_PREFIX') ?: 'SS_';
+        $prefix = $this->getDatabasePrefix();
```

Comment:
Remove this variable definition and just put `$this->getDatabasePrefix()` in its place. This will mean it's not called unnecessarily.